default/python/tox.ini

```diff
@@ -13,3 +11,6 @@ putty-ignore =
   universe_generation\universe_generator.py : +E402
   universe_generation\universe_tables.py : +E201,E122,E231
   interface_stub\__init__.py : +F401
+  # TODO: fix this warnings
+  *.py : +F812 # list comprehension redefines '<variable>' from line <line_no>
```

Comment:
I'm glad you are highlighting these, I think it would be especially good to get any F812 errors fixed right away.

Reply:
I will made it in one of the next PRs.